pages/login.page.js

```diff
@@ -4,10 +4,10 @@ let WebButton = require("../elements/button.element");
 let TextContainer = require("../elements/textContainer.element");
 
 let EC = protractor.ExpectedConditions;
+let registerLinkLocator = by.xpath('/html/body/div[1]/div[1]/div/div[1]/div[3]/p/a');
```

Comment:
should be `let registerLinkLocator = by.xpath('//a[@href="/register/"]');`

Reply:
@nbabii  corrected